components/tabstrip/overview.md

```diff
@@ -83,6 +83,51 @@ The following parameters enable you to customize the appearance of the Blazor Ta
 | `Width`   | `string` | The width of the component. You can set the Width parameter to any of the [supported units]({%slug common-features/dimensions%}). |
 | `Height`  | `string` | The height of the Component. You can set the `Height` parameter to any of the [supported units]({%slug common-features/dimensions%}). |
 
+## TabStrip Reference and Methods
+
+The `TabStrip` methods are accessible through its reference.
+
+| Method | Description |
+| --- | --- |
+| `Refresh` | Redraws the component. |
```

Comment:
We can mention what is the benefit of this method. For example, the alternative is to call `StateHasChanged()` on the parent, which will achieve the same result, but will re-render more content, and not just the TabStrip.